assets/stylesheets/_layout.scss

```diff
@@ -0,0 +1,26 @@
+// ----- Flex-box mixin (work in progress)
+@mixin grid($flex, $direction:column) {
+  flex-direction: $direction;
+  display:flex;
+  
+  & > li {      
+	  flex:$flex;
+    list-style:none;
+    margin:0 1em 0 0;
+    padding: 1em;
+  }
+  
+  @media all and (min-width: 480px) {
+  & {flex-direction: row;}  
+  }
+}
+
```

Comment:
For me, I always use `ul ` for layouts...but I understand your concern. I'm having trouble coming up with a mixin that only selects the direct child and **not** the element decedents. Do you know a solution for this?